added include_attachments

MoisesRT opened the pull request:
I added an option to include attachments while leaving intact for backwards compatibility.

blag commented:
I don't know that we need to make this a config option, just start including them.

Adding payload keys shouldn't break anything. However, if we do add the `attachments` payload key then we should bump at least the minor version number of this pack.

@LindsayHill What do you think?

MoisesRT commented:
this PR might have merge conflicts with my other PR so if its gets approved and merged ill fix it here and then update the version.

regarding adding always I agree, I did it as backwards compatible but I also agree.

MoisesRT commented:
I modified the include attachments to be included always also merged with allow_bot_messages and updated the bumped the version
@blag 
@LindsayHill

blag commented on sensors/slack_sensor.py:
```diff
@@ -158,6 +162,13 @@ def _handle_message(self, data):
+        else:
+            # Note: We resolve user and channel information to provide more context
+            user_info = self._get_user_info(user_id=data['user'])
+            
```
This line contains whitespace - that's why the build failed: https://circleci.com/gh/StackStorm-Exchange/stackstorm-slack/110#action-104

Please remove it.

MoisesRT replied:
wired, its like that master too. its not a line I changed
https://github.com/StackStorm-Exchange/stackstorm-slack/blob/master/sensors/slack_sensor.py#L171

MoisesRT replied:
Done

blag commented:
Thanks!
